packages/health-check/src/utils.js

```diff
@@ -132,6 +154,92 @@ function isPortalModuleEnabled(module) {
   return process.env.PORTAL_MODULES && process.env.PORTAL_MODULES.indexOf(module) !== -1;
 }
 
+// sleep is a helper method of sleeping for for given time. The input time is
+// expected in seconds
+async function sleep(seconds) {
+  return new Promise((r) => setTimeout(r, seconds * SECOND));
+}
+
+// skylinkHealthCheck checks if the skylink has reached full redundancy
+async function skylinkHealthCheck(skylink, numRetries = 30, authCookie, isLarge = false) {
```

Comment:
I like `attemptsLeft`. That makes it clearer to understand `attemptsLeft == 0` later because `numRetries == 0` makes it seem like we are in the first iteration.